src/stories/DropdownMenu.stories.tsx

```diff
@@ -52,3 +53,32 @@ export function FavoriteColorStory(): JSX.Element {
   )
 }
 FavoriteColorStory.storyName = 'Favorite Color'
+
+export function ExternalAnchorStory(): JSX.Element {
+  const items = React.useMemo(() => [{text: '🔵 Cyan'}, {text: '🔴 Magenta'}, {text: '🟡 Yellow'}], [])
+  const [selectedItem, setSelectedItem] = React.useState<ItemInput | undefined>()
+  const anchorRef = React.useRef<HTMLDivElement>(null)
+  const [open, setOpen] = React.useState(false)
+
+  return (
+    <Box display="flex" flexDirection="column" alignItems="flex-start">
+      <Box display="flex" flexDirection="row">
+        <DropdownButton onClick={() => setOpen(true)}>Click me to open the dropdown</DropdownButton>
+        <Box ref={anchorRef} bg="papayawhip" p={4} ml={40} borderRadius={2} display="inline-block">
+          Anchored on me!
+        </Box>
+      </Box>
+      <DropdownMenu
+        renderAnchor={null}
```

Comment:
Is explicitly setting `renderAnchor` to `null` required in order to use `anchorRef`? If so, I can see that being a confusing API for consumers.